src/continuous_agg.c

```diff
@@ -405,37 +405,68 @@ ts_continuous_agg_find_by_rv(const RangeVar *rv)
 	return ts_continuous_agg_find_by_relid(relid);
 }
 
+static ObjectAddress
+get_rel_by_name(Name schema, Name name, LOCKMODE mode)
```

Comment:
Rename as lock_rel_by_name as  that is the primary function here.

Reply:
I don't think this is accurate, because the relation might not exists. In such case semantically it is more close to the get (or get and lock if exists) operation and it returns its address too. . Maybe `get_and_lock_rel_by_name`? not fan of long names

Reply:
yes, get_and_lock_rel_by_name sounds good.